pkg/utils/iptables.go

```diff
@@ -27,49 +27,46 @@ func init() {
 	hasWait = strings.Contains(string(cmdOutput), "wait")
 }
 
-// SaveInto calls `iptables-save` for given table and stores result in a given buffer.
-func SaveInto(table string, buffer *bytes.Buffer) error {
-	path, err := exec.LookPath("iptables-save")
+// iptablesExec contains the common logic for calling save and restore
+// commands.
+func iptablesExec(cmdName string, args []string, data []byte, stdoutBuffer *bytes.Buffer) error {
+	path, err := exec.LookPath(cmdName)
 	if err != nil {
 		return err
 	}
 	stderrBuffer := bytes.NewBuffer(nil)
-	args := []string{"iptables-save", "-t", table}
 	cmd := exec.Cmd{
 		Path:   path,
 		Args:   args,
-		Stdout: buffer,
```

Comment:
The args here needs to contain the command name here as well.

So you'll need to do some work like the following in order for it to work:
```
trueArgs := append([]string{cmdName}, args...)
```

Reply:
Whoops, sorry for that!. Should be fixed now.